data/genesis/demo-marketplace.toml

```diff
@@ -0,0 +1,43 @@
+base_version = "0.2"
+upgrade_version = "0.3"
+
+[stake_table]
+capacity = 10
+
+[chain_config]
+chain_id = 999999999
+base_fee = '0 wei'
+max_block_size = '1mb'
+fee_recipient = '0x0000000000000000000000000000000000000000'
+fee_contract = '0xa15bb66138824a1c7167f5e85b957d04dd34e468'
+
+[header]
+timestamp = "1970-01-01T00:00:00Z"
+
+[[upgrade]]
+version = "0.2"
+start_proposing_view = 5
+stop_proposing_view = 15
+
+[upgrade.fee]
```

Comment:
L17 to L29 should be removed

Reply:
f8955d1f5b286330339dea212f03926e48461c5b